homeassistant/components/entur_public_transport/sensor.py

```diff
@@ -116,10 +143,10 @@ def __init__(self, api):
         """Initialize the proxy."""
         self._api = api
 
-    @Throttle(SCAN_INTERVAL)
-    def update(self) -> None:
+    @Throttle(timedelta(seconds=15))
```

Comment:
SCAN_INTERVAL ?

Reply:
Earlier the update interval was locked to the `SCAN_INTERVAL` thus the user couldn't adjust the update frequency themselves. With this I'm trying set a lowest sensible update throttling on the API without getting close to the rate limiting, for those who what more often, and a good normal update interval in the `SCAN_INTERVAL` under normal operations.
If that makes sense?

Reply:
Yes